app/dashboard/templates/legal/privacy.html

```diff
@@ -1 +1,6 @@
+  <meta name="description" content="Gitcoin Privacy Policy" />
+  <meta content="text/html; charset=UTF-8" http-equiv="content-type">
+  <style type="text/css">.lst-kix_list_14-1>li:before{content:"o  "}.lst-kix_list_14-3>li:before{content:"\0025cf  "}ul.lst-kix_list_1-0{list-style-type:none}.lst-kix_list_14-0>li:before{content:"\0025cf  "}.lst-kix_list_14-4>li:before{content:"o  "}.lst-kix_list_14-5>li:before{content:"\0025aa  "}.lst-kix_list_14-7>li:before{content:"o  "}.lst-kix_list_14-6>li:before{content:"\0025cf  "}ul.lst-kix_list_9-3{list-style-type:none}ul.lst-kix_list_9-4{list-style-type:none}ul.lst-kix_list_9-1{list-style-type:none}ul.lst-kix_list_9-2{list-style-type:none}ul.lst-kix_list_9-7{list-style-type:none}ul.lst-kix_list_9-8{list-style-type:none}ul.lst-kix_list_9-5{list-style-type:none}ul.lst-kix_list_9-6{list-style-type:none}ul.lst-kix_list_1-3{list-style-type:none}ul.lst-kix_list_1-4{list-style-type:none}ul.lst-kix_list_1-1{list-style-type:none}ul.lst-kix_list_1-2{list-style-type:none}ul.lst-kix_list_1-7{list-style-type:none}ul.lst-kix_list_9-0{list-style-type:none}ul.lst-kix_list_1-8{list-style-type:none}ul.lst-kix_list_1-5{list-style-type:none}.lst-kix_list_14-2>li:before{content:"\0025aa  "}ul.lst-kix_list_1-6{list-style-type:none}.lst-kix_85mwzbeo08jy-0>li:before{content:"\0025cf  "}.lst-kix_85mwzbeo08jy-1>li:before{content:"\0025cb  "}.lst-kix_85mwzbeo08jy-2>li:before{content:"\0025a0  "}.lst-kix_85mwzbeo08jy-4>li:before{content:"\0025cb  "}.lst-kix_85mwzbeo08jy-3>li:before{content:"\0025cf  "}.lst-kix_85mwzbeo08jy-5>li:before{content:"\0025a0  "}.lst-kix_85mwzbeo08jy-8>li:before{content:"\0025a0  "}.lst-kix_85mwzbeo08jy-7>li:before{content:"\0025cb  "}.lst-kix_85mwzbeo08jy-6>li:before{content:"\0025cf  "}.lst-kix_list_14-8>li:before{content:"\0025aa  "}.lst-kix_list_5-0>li:before{content:"\0025cf  "}.lst-kix_list_5-3>li:before{content:"\0025cf  "}.lst-kix_list_5-2>li:before{content:"\0025aa  "}.lst-kix_list_5-1>li:before{content:"o  "}ul.lst-kix_85mwzbeo08jy-0{list-style-type:none}.lst-kix_list_5-7>li:before{content:"o  "}ul.lst-kix_85mwzbeo08jy-4{list-style-type:none}ul.lst-kix_list_8-4{list-style-type:none}ul.lst-kix_85mwzbeo08jy-3{list-style-type:none}ul.lst-kix_list_8-5{list-style-type:none}.lst-kix_list_5-6>li:before{content:"\0025cf  "}.lst-kix_list_5-8>li:before{content:"\0025aa  "}ul.lst-kix_85mwzbeo08jy-2{list-style-type:none}ul.lst-kix_list_8-2{list-style-type:none}ul.lst-kix_85mwzbeo08jy-1{list-style-type:none}ul.lst-kix_list_8-3{list-style-type:none}ul.lst-kix_85mwzbeo08jy-8{list-style-type:none}ul.lst-kix_list_8-8{list-style-type:none}ul.lst-kix_85mwzbeo08jy-7{list-style-type:none}ul.lst-kix_85mwzbeo08jy-6{list-style-type:none}ul.lst-kix_list_8-6{list-style-type:none}ul.lst-kix_85mwzbeo08jy-5{list-style-type:none}ul.lst-kix_list_8-7{list-style-type:none}.lst-kix_list_5-4>li:before{content:"o  "}.lst-kix_list_5-5>li:before{content:"\0025aa  "}ul.lst-kix_list_8-0{li
```

Comment:
@mbeacom should we eventually indent this file ? 🤔

Reply:
I thought of it, but I think I will handle that in a separate issue. 😅